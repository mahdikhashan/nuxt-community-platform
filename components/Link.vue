<template>
  <div class="px-2 rounded-md hover:bg-blue-600 flex items-center justify-center text-white">
    <nuxt-link :to="address"><slot></slot></nuxt-link>
  </div>
</template>

<script>
export default {
  name: "Link",
  props: {
    address: {
      type: String,
      default: "/",
    },
  },
};
</script>

<style></style>
