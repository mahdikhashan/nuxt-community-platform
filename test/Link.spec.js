import { mount } from '@vue/test-utils'
import Link from '@/components/Tag.vue'

describe('Link', () => {
  test('is a Vue instance', () => {
    const wrapper = mount(Link)
    expect(wrapper.vm).toBeTruthy()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
