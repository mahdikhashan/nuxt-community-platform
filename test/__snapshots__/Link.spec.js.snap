// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link is a Vue instance 1`] = `
"<button>
  <nuxt-link to=\\"/\\" class=\\"flex bg-gray-50 hover:text-gray-600 hover:border-gray-600 text-gray-500 border-[1px] border-gray-400 px-3 text-[11px] justify-center items-center py-[1px] rounded-full\\">
    <div></div>
  </nuxt-link>
</button>"
`;
